packages/workbox-background-sync/test/browser/request-manager.js

```diff
@@ -14,15 +14,15 @@
 */
 
 /* eslint-env mocha, browser */
-/* global chai */
+/* global sinon, workbox */
```

Comment:
Sinon should not be needed with the work to move everything into a single eslint file.